specification/behaviors.md

```diff
@@ -0,0 +1,51 @@
+# Behaviors
```

Comment:
1. Can you add some overall description e.g. like

```
This document describes the specific behaviors for Splunk GDI Components.
```

2. What are other examples of behaviors? RUM?

3. Could we add a reference to this document in the main `README.md`?